modules/internal/ChapelBase.chpl

```diff
@@ -1907,7 +1919,18 @@ module ChapelBase {
   inline operator +=(ref lhs:imag(?w), rhs:imag(w)) {
     __primitive("+=", lhs, rhs);
   }
-  inline operator +=(ref lhs, rhs) {
+  // this one is just here so we can use isNumericType below
```

Comment:
If `isNumericType` is supposed to refer to the `+=` version right below, I think it would be nice to at least describe this as "so we can use !isNumericType(t) below", or something.